src/app/modules/home/home.html

```diff
@@ -23,7 +23,7 @@ <h1 ng-bind="::$ctrl.appName"></h1>
             <i class="fa fa-th fa-4x"></i>
          </div>
          <div class="col-md-1" ng-mouseover="$ctrl.showInfo = 5">
-            <img src="images/symbol.jpg" style="width: 4em;"></i>
+            <img src="images/nem-logo.png" style="width: 4em;"></i>
```

Comment:
maybe we should use a different logo to indicate SN program vs regular NEM?

Reply:
sure, I think that makes sense.